### PR TITLE
it not really clear

### PR DESCRIPTION
**Pull request by ramico**
i think thats how you get the value
which will 0 if not set

in mmethod save_settings line 176 in wc_sci,,_admin save the inputs

i mimiced this line of code
( self::$threshold_bad, 60 ); i suppose 60 is defaault

no documentation fro get_options

### CODE REVIEW

File: includes/class-wc-siftscience-options.php [hunk 1]
```diff
@@ -89,6 +89,10 @@ public function send_on_create_enabled() {
 			return ( get_option( self::$send_on_create_enabled ) === 'yes' );
 		}
 
+		public function get_min_order_allowence() {
```
Comment (nabsul) [inline includes/class-wc-siftscience-options.php:92]: Maybe `get_min_order_to_send()` or simply `get_min_order_value()`

**Comment by nabsul** [comment]
Here is the documentation for get_option: https://developer.wordpress.org/reference/functions/get_option/

File: includes/class-wc-siftscience-options.php (hunk 1 above)
Comment (ramico) [reply includes/class-wc-siftscience-options.php:92]: pick one

Comment (nabsul) [reply includes/class-wc-siftscience-options.php]: get_min_order_value

**Pull request merged**